### PR TITLE
Init 22.10 changelog [skip ci]

### PR DESCRIPTION
**Pull request by pxLi**
Signed-off-by: Peixin Li <pxli@nyu.edu>

Initiate 22.10 changelog w/ 
```
GITHUB_TOKEN=<MY_PAT> scripts/generate-changelog --releases=22.02,22.04,22.06,22.08,22.10
```

note: we will do a final changelog update before we build the release candidate

### CODE REVIEW

**Comment by pxLi** [comment]
@sameerz @GaryShen2008 Please help take a look at the changelog of 22.10, I will make an update after your reviews, thanks!

**Comment by sameerz** [comment]
I made only minor updates, the changelog looks good to me.

**Comment by pxLi** [comment]
build

File: CHANGELOG.md [hunk 1]
```diff
@@ -136,6 +443,8 @@ Generated on 2022-08-20
 ### PRs
 |||
 |:---|:---|
+|[#6421](https://github.com/NVIDIA/spark-rapids/pull/6421)|Fix notebook and getting started examples [skip ci]|
```
Comment (GaryShen2008) [inline CHANGELOG.md:446]: Why this PR is included in 22.08?

Comment (pxLi) [reply CHANGELOG.md:446]: image

it was marked as 22.08 PR

**Comment by pxLi** [comment]
build

**Review by GaryShen2008 — APPROVED** [review]

**Review by GaryShen2008 — APPROVED** [review]

**Comment by GaryShen2008** [comment]
build

**Pull request merged**